sources/model/managars/GGHTTPClientManager.m

```diff
@@ -281,6 +266,16 @@ - (void)useCustomer:(GGCustomer * _Nullable)customer{
     self.customer = customer;
 }
 
```

Comment:
do you mean the `setLastEventDate:`?
there is one in the ggrealtimemonitor (realtime)
and there is one for the httpManager
its not a duplicate as the tracker manager wants to know, regardless of source when any kind of event was received